dll/win32/cryptui/main.c

```diff
@@ -4219,7 +4235,9 @@ static UINT CALLBACK hierarchy_callback(HWND hwnd, UINT msg,
     {
     case PSPCB_RELEASE:
         data = (struct hierarchy_data *)page->lParam;
+#ifdef __REACTOS__
```

Comment:
same question here

Reply:
This code is not in wine.

Reply:
No, it is:
https://source.winehq.org/git/wine.git/blob/8c74027ba60a360f24cf232ee233e71262d8f1ff:/dlls/cryptui/main.c#l4222

Reply:
Sorry. Fixed.